src/backend/metal/src/lib.rs

```diff
@@ -176,50 +186,83 @@ impl core::Device for Device {
+                                                   }, // buffer
+                                                   |_, _s| { //shader
                 /*(*s.object).Release();
                 (*s.reflection).Release();*/
             },
-            |_, _p| {
-                //if !p.vs.is_null() { p.vs.release(); }
-                //if !p.ps.is_null() { p.ps.release(); }
-            }, //program
-            |_, _v| { //PSO
+                                                   |_, _p| {
+                                                       // if !p.vs.is_null() { p.vs.release(); }
+                                                       // if !p.ps.is_null() { p.ps.release(); }
+                                                   }, // program
+                                                   |_, _v| { //PSO
                 /*type Child = *mut winapi::ID3D11DeviceChild;
                 (*v.layout).Release();
                 (*(v.rasterizer as Child)).Release();
                 (*(v.depth_stencil as Child)).Release();
                 (*(v.blend as Child)).Release();*/
             },
-            |_, _v| { /*(*(v.0).0).release();*/ },  //texture
-            |_, _v| { /*(*v.0).Release();*/ }, //SRV
-            |_, _| {}, //UAV
-            |_, _v| { /*(*v.0).Release();*/ }, //RTV
-            |_, _v| { /*(*v.0).Release();*/ }, //DSV
-            |_, _v| { /*v.sampler.release();*/ }, //sampler
-            |_, _| {}, //fence
-        );
+                                                   |_, _v| {
+                                                       // (*(v.0).0).release();
+                                                   }, // texture
+                                                   |_, _v| {
+                                                       // (*v.0).Release();
+                                                   }, // SRV
+                                                   |_, _| {}, // UAV
+                                                   |_, _v| {
+                                                       // (*v.0).Release();
+                                                   }, // RTV
+                                                   |_, _v| {
+                                                       // (*v.0).Release();
+                                                   }, // DSV
+                                                   |_, _v| {
+                                                       // v.sampler.release();
+                                                   }, // sampler
+                                                   |_, _| {
+                                                       // fence
+                                                   },
+                                                   |_, _| {
+                                                       // raw mapping
+                                                   });
```

Comment:
Yeah there's some choices in rustfmt that I am kind of grossed out by, but personally I kind of think just agreeing on one standard kind of makes sense, even if you have to make some compromises on taste :P